internal/connector/internal/services/authz/authz_service.go

```diff
@@ -148,6 +148,10 @@ func (u *ValidationUser) AuthorizedNamespaceUser(errorCode errors.ServiceErrorCo
 		if u.err != nil {
 			err = u.err
 		} else {
+			if u.IsAdmin() {
```

Comment:
This is wrong. User's tenancy has to be verified first, even for admins. In other words, admin user MUST belong to the namespace org.

Reply:
Fixed with [353540f](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1365/commits/353540f5ca760f41e21353e687834f1912982304)

Reply:
@dhirajsb per my understanding this has been removed from the PR, can you please have a look and confirm all is good authentication side?